name: train
run_id: commands[0]
env COMSPEC: C:\WINDOWS\system32\cmd.exe
env NUMBER_OF_PROCESSORS: 12
env PATHEXT: .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
env PROCESSOR_ARCHITECTURE: AMD64
env PROGRAMDATA: C:\ProgramData
env PROGRAMFILES: C:\Program Files
env PROGRAMFILES(X86): C:\Program Files (x86)
env SYSTEMDRIVE: C:
env SYSTEMROOT: C:\WINDOWS
env TEMP: C:\Users\DELL\AppData\Local\Temp
env TMP: C:\Users\DELL\AppData\Local\Temp
env USERPROFILE: C:\Users\DELL
env LANG: en_US.UTF-8
env PATH: D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\.tox\test_package\Scripts;C:\Users\DELL\miniconda3;C:\Users\DELL\miniconda3\Library\mingw-w64\bin;C:\Users\DELL\miniconda3\Library\usr\bin;C:\Users\DELL\miniconda3\Library\bin;C:\Users\DELL\miniconda3\Scripts;C:\Users\DELL\miniconda3\bin;C:\Users\DELL\miniconda3\condabin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\dotnet;C:\Program Files\Microsoft SQL Server\150\Tools\Binn;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\WINDOWS\System32\OpenSSH;C:\Program Files\Git\cmd;C:\Program Files\nodejs;C:\Users\DELL\AppData\Local\Microsoft\WindowsApps;C:\Users\DELL\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\DELL\AppData\Roaming\npm
env PIP_DISABLE_PIP_VERSION_CHECK: 1
env PYTHONIOENCODING: utf-8
env PYTHONPATH: .
env PYTHONHASHSEED: 0
env TOX_ENV_NAME: train
env TOX_WORK_DIR: D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\.tox
env TOX_ENV_DIR: D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\.tox\test_package
env VIRTUAL_ENV: D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\.tox\test_package
env COLUMNS: 162
env LINES: 29
metadata pid: 23580
cwd: D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package
allow: D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\.tox\test_package\Scripts\*
cmd: python classification_model/train_pipeline.py
exit_code: 1

standard error:
Traceback (most recent call last):
  File "D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\classification_model\train_pipeline.py", line 2, in <module>
    from config.core import config
  File "D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\classification_model\config\core.py", line 7, in <module>
    import classification_model
  File "D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\classification_model\__init__.py", line 3, in <module>
    from classification_model.config.core import PACKAGE_ROOT, config
  File "D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\classification_model\config\core.py", line 87, in <module>
    config = create_and_validate_config()
  File "D:\AI_DS\Quarter_3\MLops\Practice\MLOps\Assignment_2_Production_Model_Package\classification_model\config\core.py", line 80, in create_and_validate_config
    app_config=AppConfig(**parsed_config.data),
  File "pydantic\main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 2 validation errors for AppConfig
train_data_file
  field required (type=value_error.missing)
raw_data_file
  field required (type=value_error.missing)
